Mathlib/LinearAlgebra/TensorProduct/Tower.lean

```diff
@@ -450,3 +450,76 @@ end CommSemiring
 end AlgebraTensorModule
 
 end TensorProduct
+
+namespace LinearMap
+
+open TensorProduct
+
+variable {R M₁ M₂ M₃ : Type*} (A : Type*) [CommSemiring R] [Semiring A] [Algebra R A]
+  [AddCommMonoid M₁] [Module R M₁] [AddCommMonoid M₂] [Module R M₂] [AddCommMonoid M₃] [Module R M₃]
+  (f : M₁ →ₗ[R] M₂) (g : M₂ →ₗ[R] M₃)
+
+/-- If `A` is an `R`-algebra, any `R`-linear map `M₁ → M₂` naturally induces an `A`-linear map
+`A ⊗ M₁ → A ⊗ M₂`. -/
+def extendScalars : A ⊗[R] M₁ →ₗ[A] A ⊗[R] M₂ :=
```

Comment:
To my shame I have just discovered that this duplicates [LinearMap.baseChange](https://github.com/leanprover-community/mathlib4/blob/244c4fda94ef8e27711f604be5b3a20ae5786731/Mathlib/RingTheory/TensorProduct.lean#L68).

I will follow up with a PR to remove this definition (and probably expand the comment on `LinearMap.baseChange` to include the words "extension of scalars").

Reply:
WIP here #8617